corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -272,6 +274,40 @@ hqDefine("data_dictionary/js/data_dictionary", [
             self.saveButton.setState('saved');
         };
 
+        self.newPropertyName.subscribe(function(newValue) {
+            if (!newValue) {
+                self.newPropertyNameUnique(true);
```

Comment:
It might be simpler to model this as a [computed observable](https://knockoutjs.com/documentation/computedObservables.html). e.g.
```js
self.isPropertyNameUnique = ko.computer(function() {
     // If unique condition
    return true
    // if not unique
    return false
});
```

Reply:
Thanks for the suggestion, I have changed them over to computed observables.